src/EditorFeatures/CSharpTest/MakeLocalFunctionStatic/MakeLocalFunctionStaticTests.cs

```diff
@@ -176,6 +176,187 @@ static async Task<int> fibonacci(int n)
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        [InlineData("")]
+        [InlineData("\r\n")]
+        [InlineData("\r\n\r\n")]
+        public async Task TestLeadingTriviaAfterOpenBrace(string leadingTrivia)
+        {
+            await TestInRegularAndScriptAsync(
+$@"using System;
+
+class C
+{{
+    void M()
+    {{{leadingTrivia}
+        int [||]fibonacci(int n)
+        {{
+            return n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+        }}
+    }}
+}}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        static int fibonacci(int n)
+        {
+            return n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+        }
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        [InlineData("")]
+        [InlineData("\r\n")]
+        [InlineData("\r\n\r\n")]
+        public async Task TestLeadingTriviaAfterLocalFunction(string leadingTrivia)
+        {
+            await TestInRegularAndScriptAsync(
+$@"using System;
+
+class C
+{{
+    void M()
+    {{
+        bool otherFunction()
+        {{
+            return true;
+        }}{leadingTrivia}
+        int [||]fibonacci(int n)
+        {{
+            return n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+        }}
+    }}
+}}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        bool otherFunction()
+        {
+            return true;
+        }
+
+        static int fibonacci(int n)
+        {
+            return n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+        }
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        [InlineData("")]
+        [InlineData("\r\n")]
+        [InlineData("\r\n\r\n")]
+        public async Task TestLeadingTriviaAfterExpressionBodyLocalFunction(string leadingTrivia)
+        {
+            await TestInRegularAndScriptAsync(
+$@"using System;
+
+class C
+{{
+    void M()
+    {{
+        bool otherFunction() => true;{leadingTrivia}
+        int [||]fibonacci(int n) => n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+    }}
+}}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        bool otherFunction() => true;
+
+        static int fibonacci(int n) => n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        [InlineData("")]
+        [InlineData("\r\n")]
+        [InlineData("\r\n\r\n")]
+        public async Task TestLeadingTriviaAfterComment(string leadingTrivia)
+        {
+            await TestInRegularAndScriptAsync(
+$@"using System;
+
+class C
+{{
+    void M()
+    {{
+        //Local function comment{leadingTrivia}
```

Comment:
💡 It would be good to have one more where the blank lines precede the comment

Reply:
Were you thinking something like this?

```csharp
void M()
{{{leadingTrivia}
   //Comment
   void localFunction() ...
```